common/predictive-text/worker/worker-interfaces.ts

```diff
@@ -118,7 +118,7 @@ interface LMLayerWorkerState {
  */
 interface WorkerInternalModel {
   configure(capabilities: Capabilities): Configuration;
-  predict(transform: Transform, context: Context): Suggestion[];
+  predict(transform: Transform, context: Context): Distribution<Suggestion>;
```

Comment:
Idea: do not change the signature of `predict()`; instead, add `predictWithDistribution()` that has this same method signature.

Then move all of the code in `predict()` of existing models in this PR to `predictWithDistribution()`. `predict()` will delegate to `predictWithDistribution()` and will "unwrap" the distribution: essentially all of that `.map(function(value) { return value.sample })` stuff.